src/pages/Landing.tsx

```diff
@@ -70,6 +70,27 @@ const SubText = styled.h3`
   }
 `
 
+const CTAButton = styled(BaseButton)`
+  width: 288px;
+  padding: 16px;
+  border-radius: 24px;
+  color: ${({ theme }) => theme.white};
+
+  &:hover {
+    opacity: 50%;
+  }
+`
+
+const ButtonCTA = styled(CTAButton)`
+  background: linear-gradient(10deg, rgba(255, 0, 199, 1) 0%, rgba(255, 159, 251, 1) 100%);
```

Comment:
Does it make sense to reference these values from theme, instead of hardcoding here? Sounds like we have all shades of pink available already

Reply:
i can look into changing the colors here